### PR TITLE
fix #138 add removeAllSwipeListner function 

### LINKED ISSUE #138: Remove all Swipe Listeners
It will be very useful for me to have a removeAllSwipeListeners() method like the removeAllSwipeDeniers() one.

### PR DESCRIPTION
**Pull request by daimajia**
BTW, prettify the code.

### CODE REVIEW

**Comment by daimajia** [comment]
@jpshelley @linfaxin when available, please check it.

**Comment by daimajia** [comment]
It's a very simple change.

File: library/src/main/java/com/daimajia/swipe/SwipeLayout.java [hunk 1]
```diff
@@ -1508,22 +1557,22 @@ public void setBottomViewIds(int leftId, int rightId, int topId, int bottomId) {
...
 
     private void updateBottomViews() {
         View currentBottomView = getCurrentBottomView();
-        if(currentBottomView!=null){
+        if (currentBottomView != null) {
             if (mCurrentDragEdge == DragEdge.Left || mCurrentDragEdge == DragEdge.Right) {
                 mDragDistance = currentBottomView.getMeasuredWidth() - dp2px(getCurrentOffset());
-            }
-            else mDragDistance = currentBottomView.getMeasuredHeight() - dp2px(getCurrentOffset());
+            } else
```
Comment (kostartur) [inline library/src/main/java/com/daimajia/swipe/SwipeLayout.java]: missing brackets
``` } else {

    }```

File: library/src/main/java/com/daimajia/swipe/SwipeLayout.java [hunk 2]
```diff
@@ -703,31 +723,31 @@ public void addView(View child, int index, ViewGroup.LayoutParams params) {
             e.printStackTrace();
         }
 
-        if(gravity>0){
+        if (gravity > 0) {
             gravity = GravityCompat.getAbsoluteGravity(gravity, ViewCompat.getLayoutDirection(this));
 
-            if((gravity & Gravity.LEFT) == Gravity.LEFT){
+            if ((gravity & Gravity.LEFT) == Gravity.LEFT) {
```
Comment (kostartur) [inline library/src/main/java/com/daimajia/swipe/SwipeLayout.java]: (gravity & Gravity.LEFT) != 0 would do

**Pull request closed without merging**